ownCloud/Resources/Theming/Branding.plist

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
+	<key>authentication.browser-session-class</key>
+	<string>MIBrowser</string>
```

Comment:
Example should be removed. I will put the sample code inside the PR comment.

Reply:
Done. Turned the the plist into an example at `ownCloud/Resources/Theming/Branding-mibrowser-example.plist`.